basic/basic.html

```diff
@@ -15,6 +15,12 @@
 
     <hr />
 
+    <div class="content-row">
+      <input type="text" class="input2" />
```

Comment:
nazwa powinna bardziej wyjaśniać co to za element, np. filter-input, filter-field, etc.

Reply:
nazwa zmieniona na filter_input

Reply:
zamiast _ użyj - w nazwach klas